tests/testthat/test-infection-integration.R

```diff
@@ -855,62 +893,62 @@ test_that('calculate_treated() returns an empty Bitset when the parameter list c
+                                                 coverages = round(runif(1, 0, 1/2),
+                                                                   digits = 2))
+            parameters <- set_clinical_treatment(parameters = parameters,
+                                                 drug = 2,
+                                                 timesteps = 1,
+                                                 coverages = round(runif(1, 0, 1/2),
+                                                                   digits = 2))
+            parameters <- set_antimalarial_resistance(parameters = parameters,
+                                                      drug = 1,
+                                                      timesteps = 1,
+                                                      artemisinin_resistance_proportion = 1,
+                                                      partner_drug_resistance_proportion = 0,
+                                                      slow_parasite_clearance_probability = 0,
+                                                      early_treatment_failure_probability = 1,
+                                                      late_clinical_failure_probability = 0,
+                                                      late_parasitological_failure_probability = 0,
+                                                      reinfection_during_prophylaxis_probability = 0, 
+                                                      slow_parasite_clearance_time = 10)
+            parameters <- set_antimalarial_resistance(parameters = parameters,
+                                                      drug = 2,
+                                                      timesteps = 1,
+                                                      artemisinin_resistance_proportion = 1,
+                                                      partner_drug_resistance_proportion = 0,
+                                                      slow_parasite_clearance_probability = 0,
+                                                      early_treatment_failure_probability = 1,
+                                                      late_clinical_failure_probability = 0,
+                                                      late_parasitological_failure_probability = 0,
+                                                      reinfection_during_prophylaxis_probability = 0, 
+                                                      slow_parasite_clearance_time = 20)
+            
+            clinical_infections <- individual::Bitset$new(100)
+            clinical_infections$insert(sample.int(n = 100, size = round(runif(n = 1, min = 10, max = 100)), replace = FALSE))
+            timestep <- 5
+            events <- create_events(parameters)
+            variables <- create_variables(parameters = parameters)
+            renderer <- individual::Render$new(timesteps = 10)
+            
+            treated <- calculate_treated(variables = variables,
```

Comment:
Could we fix the indentation here to match the other tests? It could be a tab vs spaces thing

Reply:
Good spot - I've fixed the indentation so that it is consistent between all tests